### PR TITLE
Adding new files

### PR DESCRIPTION
**Pull request by rostyslav-kuchmynda**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+.info_panel {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: .5rem;
+}
+
+.info_image {
+    padding: .5rem;
+    background-color: #ffffff;
```
Comment (karmeljuk) [inline css/style.css]: optimize color

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+}
+
+img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
```
Comment (karmeljuk) [inline css/style.css]: optimize margins

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    padding: 1rem .5rem;
+}
+
+.stories {
+    padding: 5rem 1rem;
+}
+
+.subscribe,
+.footer {
+    padding: 1rem 1rem;
```
Comment (karmeljuk) [inline css/style.css]: optimize padding

Comment (rostyslav-kuchmynda) [reply css/style.css]: Done.

File: css/style.css (hunk 2 above)
Comment (rostyslav-kuchmynda) [reply css/style.css]: Done.

File: css/style.css (hunk 1 above)
Comment (rostyslav-kuchmynda) [reply css/style.css]: Done.

**Pull request merged**